lib/sso_users_api/all_app_users.rb

```
@@ -11,12 +11,41 @@ def initialize(user)
     end
 
     def call
+      # Check both Identity Server and Auth0 for users
+      all_users = []
+
+      # Check Identity Server
+      all_users.concat(identity_server_users)
+
+      # Check Auth0
+      all_users.concat(auth0_users)
+
+      all_users
+    rescue StandardError => e
+      NfgRestClient::Logger.error("AllAppUsers error: #{e.message}")
```

What is the reason for this rescue ?  
if there is an error won't it silently fail ?